website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-choosing-a-dbt-version.md

```diff
@@ -70,4 +73,4 @@ Additionally upgrading to more recent versions of dbt Core will enable better pe
 
 #### Need help upgrading?
 
-If you want more advice on how to upgrade your dbt projects, check out our [migration guides](docs/guides/migration-guide/upgrading-to-0-21-0) and our [upgrading Q&A page](docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions).
+If you want more advice on how to upgrade your dbt projects, check out our [migration guides](/guides/migration/versions/upgrading-to-v1.1) and our [upgrading Q&A page](docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions).
```

Comment:
perhaps the migrations landing page is more evergreen?
<img width="500" alt="image" src="https://user-images.githubusercontent.com/8158673/172759695-276b91b8-a1e3-4db2-a7d5-70de69b66038.png">

```suggestion
If you want more advice on how to upgrade your dbt projects, check out our [migration guides](/guides/migration/versions/) and our [upgrading Q&A page](docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions).
```